src/style.css

```diff
@@ -1 +1,20 @@
 /* styles go here */
+.form-field {
+  display: block;
+}
+
+input {
```

Comment:
It's better to avoid styling HTML tags directly. Instead, consider using a class for more specific targeting to avoid unintended styling side effects

Reply:
isn't fixed for `fieldset`. Style class selectors instead of tag ones